README.md

```diff
@@ -10,6 +10,30 @@ is [Zig](https://ziglang.org/download/).
 Zig API bindings are also provided via the "av" module. See `doc/examples` for
 API usage examples.
 
+## Usage
+
+```
+zig fetch --save=ffmpeg git+https://github.com/allyourcodebase/ffmpeg#main
```

Comment:
this would result in an url that looks like this:

```
+        .ffmpeg = .{
+            .url = "git+https://github.com/allyourcodebase/ffmpeg?ref=main#13f5197369abf3ca86b61933de432b4497279925",
+            .hash = "12202056dbe80cb54abece0c7c0f0a3f3db379cbfe8f9f5f58334ad3eede996be838",
+        },
```

why would you do that? just put `zig fetch --save=ffmpeg git+https://github.com/allyourcodebase/ffmpeg` and end up with:

```
+        .ffmpeg = .{
+            .url = "git+https://github.com/allyourcodebase/ffmpeg#13f5197369abf3ca86b61933de432b4497279925",
+            .hash = "12202056dbe80cb54abece0c7c0f0a3f3db379cbfe8f9f5f58334ad3eede996be838",
+        },
```

or better yet use an actual release like this:

```
        .ffmpeg = .{
            .url = "https://github.com/allyourcodebase/ffmpeg/archive/refs/tags/7.0.1-5.tar.gz",
            .hash = "12202056dbe80cb54abece0c7c0f0a3f3db379cbfe8f9f5f58334ad3eede996be838",
        },
```

This last one will download the fastest.

Reply:
For me, the build.zig.zon from `#main` is identical to the build.zig without it (with zig 0.14.0-dev.1694+3b465ebec)

` zig fetch --save=ffmpeg git+https://github.com/allyourcodebase/ffmpeg#main`:

```zig
.url = "git+https://github.com/allyourcodebase/ffmpeg#13f5197369abf3ca86b61933de432b4497279925",
```

` zig fetch --save=ffmpeg git+https://github.com/allyourcodebase/ffmpeg`:

```zig
.url = "git+https://github.com/allyourcodebase/ffmpeg#13f5197369abf3ca86b61933de432b4497279925",
```

I didn't realize it automatically resolved to the main branch without specifying. I'll switch it to using a tagged release.